.github/workflows/cicd.yml

```diff
@@ -0,0 +1,76 @@
+name: CICD
+
+on: [push]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+
+jobs:
+
+  run-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout this repository
+        uses: actions/checkout@v2
+        with:
+            lfs: true
+
+      - name: Build application container
+        shell: bash -l {0}
+        run: docker build -t materials-designer:centos -f dockerfiles/centos/Dockerfile .
+
+      - name: Build test container
+        shell: bash -l {0}
+        run: docker build -t materials-designer:test -f dockerfiles/test/Dockerfile .
+
+      - name: Run tests
+        shell: bash -l {0}
+        run: |
+          docker run -p 3001:3001 -d materials-designer:centos
+          sleep 60
```

Comment:
Is this to wait for the application to get started? We should consider using a curl command instead and a while loop (we have several examples in `saltstack` repo when waiting for services to start). There's no guarantee that the application will start within 60 seconds (b/c we are running in the cloud) and if it starts faster, we are wasting the remaining 50.

Reply:
This has been added